.drone.yml

```diff
@@ -1,13 +0,0 @@
-kind: pipeline
```

Comment:
don't think this should make a difference since we don't run Drone on system-charts at all (not even on the release branches). @cbron @PennyScissors feel free to correct me if I'm wrong!